laos_gggi/co2_processing.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import os
+
+
+def process_co2():
+    if not os.path.isfile("../data/co2.csv"):
+        url = "https://gml.noaa.gov/aftp/products/trends/co2/co2_mm_gl.csv"
+        df = pd.read_csv(url, skiprows=38)
+        df["day"] = 1
+        df["Date"] = pd.to_datetime(dict(year=df.year, month=df.month, day=df.day))
```

Comment:
No need to make the dummy `day` column, you can use `apply` together with the `format` argument of `to_datetime`:

`df["Date"] = df.apply(lambda x: pd.to_datetime(f'{x.year}-{x.month}', format='%Y-%m', axis=1)`

The axis argument is important, because it means `x` will be a row of data, rather than a column.

That format string might be wrong, I always forget what is capitalized and what is lower-case. You can google it to double-check

Reply:
Yeah this always trips me up. "m" is month and "M" is minute

Reply:
That's horrible, M should be month because months are bigger than minutes. Reminds me of this: https://youtu.be/Xto-7_xWb9g?feature=shared&t=605